pennylane/ops/meta.py

```diff
@@ -119,7 +120,7 @@ class WireCut(Operation):
     num_wires = AnyWires
     grad_method = None
 
-    def __init__(self, *params, wires=None, id=None):
+    def __init__(self, *params, wires: WiresLike = None, id=None):
         if wires == []:
             raise ValueError(
                 f"{self.__class__.__name__}: wrong number of wires. "
```

Comment:
Let me try this out. I know I had some issues with this one ...

Reply:
Yeah .... [tests](https://github.com/PennyLaneAI/pennylane/actions/runs/12424399314/job/34689492990?pr=6689) are not happy. I think because we wish to do things like `qml.WireCut(0)` and have it infer that `wires=0`. This can only be done if `wires` is a kwarg and defaulted to `None`. 😢

Reply:
Yes, this is weird, but since it is not related to this PR I would say it's fine :+1:

Reply:
Ah, I see 🤔  Does something bad happen if we remove the positional `*params`? WireCut takes 0 parameters, if anything is passed as params an error is raised in the `Operation` validation:

```
>>> qml.WireCut(1.2, 0)
ValueError: WireCut: wrong number of parameters. 1 parameters passed, 0 expected.
```

So its odd its part of the call signature. I think removing `*params` would fix that problem (and make this cleaner/more logical).

Reply:
I would say it's somewhat related to the PR, in the sense that it keeps the implementation in this PR more uniform, and its arguably part of cleaning up how we handle empty wires and `wires=None`, but I'm also happy to let it be 😅

Reply:
Let me try it out. 😃

Reply:
Good idea! Seems to work and makes a whole lot of sense to me 🙂!